### PR TITLE
Test fix speed

### PR DESCRIPTION
**Pull request by khancyr**
Correct some syntax leftover
Add epsilon to ground speed check

### CODE REVIEW

File: Tools/autotest/common.py [hunk 1]
```diff
@@ -2770,12 +2770,12 @@ def wait_altitude(self, altitude_min, altitude_max, relative=False, timeout=30,
...
-                if alt_relative:
-                    return msg.relative_alt / 1000.0  # mm -> m
-                else:
-                    return msg.alt / 1000.0  # mm -> m
-            raise MsgRcvTimeoutException("Failed to get Global Position")
+            if msg is None:
+                raise MsgRcvTimeoutException("Failed to get Global Position")
+            if alt_relative:
+                return msg.relative_alt / 1000.0  # mm -> m
+            else:
```
Comment (peterbarker) [inline Tools/autotest/common.py]: Don't need this else

**Comment by khancyr** [comment]
it prevent things like that : (want 0.000000 +- 0.200000), reach 0.200155

**Comment by peterbarker** [comment]
@khancyr sure - but where' the 0.2 come from?  Isn't that a number I just /made up/ because the thing wasn't stopping? :-)  In which case you can just raise that number.  Doing so would remove the choice of epsilon.

I'm really not fussed.  We can merge this.

**Comment by khancyr** [comment]
I think that it is still good to have an espilon, under 1cm/s value will be mostly garbage from the EKF anyway.

I have raised the value to 0.25.
I will made another PR with a variable for safe stopping speed

File: Tools/autotest/common.py (hunk 1 above)
Comment (khancyr) [reply Tools/autotest/common.py]: done

**Comment by peterbarker** [comment]
... it occurs to me I was actually using that 0,2 bound *as* an epsilon.  It might be we change the caller to 0 and have the epsilon at 0.25 instead?

**Comment by khancyr** [comment]
closed as the issue was fixed as part of another PR

**Pull request closed without merging**